packages/supercollider/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=http://supercollider.github.io/
+TERMUX_PKG_DESCRIPTION="A platform for audio synthesis and algorithmic composition"
+TERMUX_PKG_LICENSE="GPL-3.0"
```

Comment:
GPL-3.0 is accurate.
But if you want to be more specific, some parts of this project are also GPL-2.0-or-later.
```suggestion
TERMUX_PKG_LICENSE="GPL-2.0-or-later, GPL-3.0"
```